requirements.txt

```diff
@@ -1,3 +1,3 @@
 dvc[s3]==2.36.0
-torch==1.12.0
-torchvision==0.13.0
+torch==1.13.0
```

Comment:
`torch` and `torchvision` need to be updated together else pip throws an error:

```
torchvision 0.14.0 requires torch==1.13.0, but you'll have torch 1.12.0 which is incompatible.
```

Reply:
Did you run the e2e tests against this?

Reply:
> Did you run the e2e tests against this?

They're passing on my end!